medinify/scrapers/iodine_scraper.py

```diff
@@ -0,0 +1,59 @@
+''' Drug review scraper for Medinify 
```

Comment:
Please follow the Google Python styling guide for comments. Should be three double quotes, not single quotes.